custom_components/waste_collection_schedule/waste_collection_schedule/source/citiesapps_com.py

```diff
@@ -0,0 +1,67 @@
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.CitiesAppsCom import CitiesApps, SERVICE_MAP
+
+from datetime import datetime
+
+EXTRA_INFO = SERVICE_MAP
+TITLE = "App CITIES"
+DESCRIPTION = "Source for App CITIES."
+URL = "https://citiesapps.com"
+TEST_CASES = {
+    "Fürstenfeld Haushalt Altenmarkt": {
+        "city": "Fürstenfeld",
+        "calendar": "Haushalt Altenmarkt"
+    },
+    "Buch - St. Magdalena Buch - St. Magdalena": {
+        "city": "Buch - St. Magdalena",
+        "calendar": "Buch - St. Magdalena"
+    },
+    "Rudersdorf Rudersdorf 3": {
+        "city": "Rudersdorf",
+        "calendar": "Rudersdorf 3"
+    },
+    "Lieboch": {
+        "city": "Lieboch",
+        "calendar": "Lieboch"
+    }
+}
+COUNTRY = "at"
+
+
+ICON_MAP = {
+    "Restmüll": "mdi:trash-can",
+    "Biomüll": "mdi:leaf",
+    "Altpapier": "mdi:package-variant",
+    "Papier": "mdi:package-variant",
+    "Leichtfraktion": "mdi:recycle",
+    "Leichtverpackung": "mdi:recycle",
+    "Gelber": "mdi:recycle",
+    "Sonder-": "mdi:dump-truck",
+    "Abfallwirtschaftszentrum": "mdi:house",
+    "Strauchschnitt": "mdi:tree",
+}
+
+
+AUTH_TOKEN = "Basic NjQzNmI2NWFiZjBmYzkwMDEyOTdkNjU0OmQ1Y2IxMzYxNzJiODBkMmI0NTQyN2MyZDUyYTZhMTNmMDhlOGViZDE0M2FkNTVlNjI3MzZmZTY3ZjNkNw=="
```

Comment:
Isn't this the same token as in the service folder? If yes, I would import it here to use the definition from service.

Reply:
You're right this token is not needed here at all anymore